### PR TITLE
Adding GPOOL

### PR DESCRIPTION
**Pull request by mik1893**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -1727,5 +1727,20 @@
       "telegram":"manizzler",
       "operatorName":"manizzler",
       "github":"manizzle"
+   },
+   {
+      "ticker":"GPOOL",
+      "poolId":"eb6e1949ef2e1f2cdb9b041d2a4b86d4d837798ab3ddfe53cc4cafe4",
```
Comment (lophre) [inline registry.json:1733]: poolid is mapping to ADATA on PoolTool.

**Review by lophre — CHANGES_REQUESTED** [review]
Please update with proper ticker

File: registry.json (hunk 1 above)
Comment (mik1893) [reply registry.json:1733]: yes sorry i thought i updated the pool request correctly but didn't. 
We renamed the pool some days ago. Will do now

File: registry.json [hunk 2]
```diff
@@ -1727,5 +1727,20 @@
...
+      "social":{
+         "facebook":"",
+         "twitter":"https://twitter.com/adapooldata",
+         "website":"https://www.pooldata.live",
+         "reddit":"",
+         "instagram":"",
+         "youtube":""
+      },
+      "telegram":"pooldata",
```
Comment (lophre) [inline registry.json:1742]: telegram field can't be a user group.  Please provide a username as SPA voting is done using Telegram.

**Review by lophre — CHANGES_REQUESTED** [review]
Please change telegram field as per latest comment

**Comment by mik1893** [comment]
Changed telegram field

**Review by mik1893 — COMMENTED** [review]
changed

**Review by lophre — APPROVED** [review]

**Pull request merged**